docs/openapi-issuer.yaml

```diff
@@ -0,0 +1,570 @@
+openapi: 3.0.3
+info:
+  title: Firma con IO - Issuer API
+  version: 1.0.0
+servers:
+  - url: https://api.io.pagopa.it/api/v1/sign
+    description: production
+security:
+  - SubscriptionKey: []
+paths:
+  /dossiers:
+    post:
+      operationId: createDossier
+      tags:
+        - Dossier
+      summary: Create a Dossier
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/CreateDossierBody"
+      responses:
+        "201":
+          description: The Dossier detail
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/DossierDetailView"
+        "400":
+          $ref: "#/components/responses/BadRequest"
+        "403":
+          $ref: "#/components/responses/Forbidden"
+        default:
+          $ref: "#/components/responses/Unexpected"
+
+  /dossiers/{id}:
+    get:
+      operationId: getDossier
+      tags:
+        - Dossier
+      summary: Get a Dossier by Id
+      parameters:
+        - in: path
+          name: id
+          required: true
+          schema:
+            $ref: "#/components/schemas/Id"
+      responses:
+        "200":
+          description: The Dossier detail
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/DossierDetailView"
+        "403":
+          $ref: "#/components/responses/Forbidden"
+        "404":
+          $ref: "#/components/responses/NotFound"
+        default:
+          $ref: "#/components/responses/Unexpected"
+
+  /signers:
+    post:
+      operationId: getSignerByFiscalCode
```

Comment:
why is this a post and not a get?

Reply:
in order to avoid passing the fiscal code as route param

Reply:
What's the problem with passing the fiscalCode as route param?

Reply:
because the fiscal code is sensitive data (pii) and I would like to avoid "involuntary logging"